lib/tether/interfaces.go

```diff
@@ -12,37 +12,41 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package main
+package tether
 
 import (
-	"net"
-	"os"
+	"io"
 
 	"github.com/vmware/vic/lib/metadata"
-	"github.com/vmware/vic/lib/portlayer/attach"
 	"github.com/vmware/vic/pkg/dio"
-	"golang.org/x/crypto/ssh"
 	"golang.org/x/net/context"
 )
 
-// ops is here so we can switch the OS Ops impls for test mocking
-var ops osops
-var utils utilities
+type Operations interface {
+	Setup() error
+	Cleanup() error
+	// Log returns the tether debug log writer
```

Comment:
only for the tether when running in an actual container. For tests it goes into /tmp/*/dev/ttyS1 and when used in the appliance it goes to /var/log/vic/init.log